com.unity.shadergraph/Editor/GraphUI/EditorCommon/CommandStateObserver/ShaderGraphCommandOverrides.cs

```diff
@@ -255,13 +255,12 @@ RenameElementCommand renameElementCommand
             UpdateConstantValueCommand updateConstantValueCommand)
         {
             var shaderGraphModel = (ShaderGraphModel)graphModelState.GraphModel;
-            var model = updateConstantValueCommand.OwnerModel as GraphDataPortModel;
-            if (model.NodeModel is GraphDataNodeModel nodeModel && shaderGraphModel != null)
+            if (updateConstantValueCommand.Constant is ICLDSConstant cldsConstant)
```

Comment:
this is good! given that everything is routing through Constants now I imagine this should handle every use case out there, instead of checking for is this a node, is this a port, is this an inspector field etc, we can just call `UpdateConstantValueCommand` and as long as the values backed by a CLDS constant, we're good to go

Reply:
Yeah- definitely learned a ton about IConstants while working on this. Hoping to get this integrated with static ports eventually so we can get rid of the other similar commands for node parts